src/model/isAnswerIsCorrect.js

```diff
@@ -0,0 +1,2 @@
+export function isAnswerIsCorrect(correctAnswer,userAnswer)
+{return isAnswerIsCorrect.length == arguments.length && correctAnswer===userAnswer};
```

Comment:
`return correctAnswer===userAnswer`
A wpisz tylko to i zobacz czy jakiś test nie przejdzie.
Daj znać w komentarzu. Jeśli któreś testy nie przechodzą to sprawdź i wstaw które.

Reply:
poprawione ;)